src/blog_list.ts

```diff
@@ -0,0 +1,52 @@
+type Blog = {
+    title: string;
+    date: string;
+    description: string;
+    slug: string;
+}
+
+const blogList: Blog[] = [
+
+    {
+        title: "My day at work",
+        date: "October 19, 2023",
+        description: "My coworkers really were on my nerves today.",
+        slug: "/Users/ethantrantalis/Projects/bootcamp-project-2023/blog1.html"
+    },
+    {
+        title: "Going home this weekend",
+        date: "October 16, 2023",
+        description: "Driving down the US-101 is a favorite of mine, the vibes are so good.",
+        slug: "/Users/ethantrantalis/Projects/bootcamp-project-2023/blog2.html"
+    }
+];
+
+ 
+let blogContainer = document.querySelector(".blogs_added");
+
+                
+function addNewBlogs(blogs: Blog[]) {
+
+
+
+    for(let i = 0; i < blogList.length; i++){
```

Comment:
In js/ts, take advantage of built in list functions! For example, `forEach()` and `map()`